### PR TITLE
Add support for NVMe to NodePublishVolume

### PR DESCRIPTION
**Pull request by pwschuurman**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**: This PR adds support for NVMe persistent disks to GCP PD CSI Driver. Currently only SCSI disks are supported.

**Does this PR introduce a user-facing change?**:
```release-note
Add support for NVMe persistent disks
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -26,7 +26,7 @@ RUN GOARCH=$(echo $TARGETPLATFORM | cut -f2 -d '/') GCE_PD_CSI_STAGING_VERSION=$
 # Start from Kubernetes Debian base.
 FROM k8s.gcr.io/build-image/debian-base:buster-v1.9.0 as debian
 # Install necessary dependencies
-RUN clean-install util-linux e2fsprogs mount ca-certificates udev xfsprogs
+RUN clean-install util-linux e2fsprogs mount ca-certificates udev xfsprogs nvme-cli xxd bash
```
Comment (mattcary) [inline Dockerfile:29]: do we need xxd & bash, or did you just use that for debugging?

File: pkg/mount-manager/device-utils.go [hunk 2]
```diff
@@ -134,9 +145,58 @@ func parseScsiSerial(output string) (string, error) {
 	return substrings[1], nil
 }
 
+// getScsiSerial calls google_nvme_id on the given devicePath to get the serial
```
Comment (mattcary) [inline pkg/mount-manager/device-utils.go:148]: s/getScsiSerial/getNvmeSerial/g

**Comment by mattcary** [comment]
lgtm modulo a nit

File: pkg/mount-manager/device-utils.go (hunk 2 above)
Comment (pwschuurman) [reply pkg/mount-manager/device-utils.go]: Thanks for catching. Updated.

**Review by mattcary — APPROVED** [review]
/lgtm
/approve

**Pull request merged**